pdns/recursordist/docs/settings.rst

```diff
@@ -548,6 +548,40 @@ found, the recursor fallbacks to sending 127.0.0.1.
 This is the value set for the EDNS0 buffer size in outgoing packets.
 Lower this if you experience timeouts.
 
+.. _setting-edns-padding-from:
+
+``edns-padding-from``
+---------------------
+.. versionadded:: 4.4.0
+
+-  Comma separated list of netmasks
+-  Default: (none)
+
+List of netmasks (proxy IP in case of XPF or proxy-protocol presence, client IP otherwise) for which EDNS padding will be enabled in responses. See also `edns-padding-mode`_.
+
+.. _setting-edns-padding-mode:
+
+``edns-padding-mode``
+---------------------
+.. versionadded:: 4.4.0
+
+-  One of ``always``, ``padded-queries-only``, String
+-  Default: ``padded-queries-only``
+
+Whether to add EDNS padding to all responses (``always``) or only to the ones for padded queries (``padded-queries-only``, the default) coming from `edns-padding-from`_ sources.
```

Comment:
From this description it is not immediate clear if queries outside `edns-padding-from` are padded or not.

Reply:
This PR implements padding for outgoing responses, not for outgoing queries, perhaps this description should be clearer about that?

Reply:
My comment should have read:
From this description it is not immediate clear if responses to queries outside edns-padding-from are padded or not.

Reply:
I have updated the description, please let me know if it's still unclear. It's sometimes hard to take the user point-of-view when you wrote you spent some time writing the actual code! :)

Reply:
Yes, much more clear.